### ISSUE TITLE
[Roadmap] Make issue numbers clickable

### ISSUE THREAD

**Issue by Andre601** [mastodon/joinmastodon#284]
The issue numbers (i.e. `MAS-23`) should be clickable and point to the actual issues and PRs, if that is doable.

**Comment by theotherlinh** [comment]
Or some kind of reference that is searchable here?

**Comment by BenAveling** [comment]
> the links won't be available publicly 

I agree. The final, internal issue tracker is private. And actually I don't have a problem with that. They do need a place for a conversation without all of us contributing. And we don't need full visibility of internal discussion.

But I would like to see a bit more information being available to the public, rather than just, this is maybe a candidate, maybe.

Perhaps they could have a field in each linear-issue that says: this is what's happening, the topic is known to be under discuss ion in this/these github-issue/s... 

And then we can click on the issue to expand it and see the current content of that field.

I mean, I guess, any of us could start creating a github-issue for each linear-issue. But it would be better to have something endorsed by the dev team. 

Just keep us "consulted/informed", please.
